agent/initialization/start.go

```diff
@@ -1,12 +1,37 @@
 package initialization
 
 import (
+	"context"
 	"fmt"
+	"log"
 
+	"github.com/kubeshop/tracetest/agent/client"
 	"github.com/kubeshop/tracetest/agent/config"
+	"github.com/kubeshop/tracetest/agent/proto"
+	"github.com/kubeshop/tracetest/agent/workers"
 )
 
 // Start the agent with given configuration
 func Start(config config.Config) {
 	fmt.Println("Starting agent")
+	ctx := context.Background()
+
+	client, err := client.Connect(ctx, config.ServerURL,
+		client.WithAPIKey(config.APIKey),
+		client.WithAgentName(config.AgentName),
```

Comment:
is this agent name configurable by user? does it act as an ID or is it just for displaying purposes?

Reply:
only for displaying purposes. By default, it's the host name, but the user might override it with a flag: `tracetest agent start --name my-agent`